helix-core/src/main/java/org/apache/helix/controller/dataproviders/BaseControllerDataProvider.java

```diff
@@ -544,6 +544,14 @@ public synchronized void setLiveInstances(List<LiveInstance> liveInstances) {
     return _instanceMessagesCache.getMessages(instanceName);
   }
 
+  public Map<String, Map<String, Message>> getStaleMessages() {
+    return _instanceMessagesCache.getStaleMessageCache();
+  }
+
+  public void setStaleMessages(Map<String, Map<String, Message>> staleMessageMap) {
```

Comment:
nit, cacheStaleMessages so all the updating message cache methods are named in the same way?

Reply:
Actually I found in `InstanceMessageCache`, more are named with `cache`, so I changed in that file.